src/TodoList.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Todo from './Todo';
+
+const TodoList = (
+  { usersAndTodosArr, sortByTitle, sortByName, sortByProgress }
+) => (
+  <section>
+    <button
+      type="button"
+      onClick={sortByTitle}
```

Comment:
```suggestion
      onClick={sortBy('title')}
```
а в sortBy викликай метод getSortMethod (hint)